pkg/fedvolumebackup/backup/backup_manager.go

```diff
@@ -504,22 +504,23 @@ func (bm *backupManager) buildBackupMember(volumeBackupName string, clusterMembe
 			Labels:      labels,
 		},
 		Spec: pingcapv1alpha1.BackupSpec{
-			Mode:                     pingcapv1alpha1.BackupModeVolumeSnapshot,
-			Type:                     pingcapv1alpha1.BackupTypeFull,
-			FederalVolumeBackupPhase: pingcapv1alpha1.FederalVolumeBackupExecute,
-			ResourceRequirements:     *backupTemplate.ResourceRequirements.DeepCopy(),
-			Env:                      backupTemplate.Env,
-			BR:                       backupTemplate.BR.ToBRMemberConfig(clusterMember.TCName, clusterMember.TCNamespace),
-			StorageProvider:          *backupTemplate.StorageProvider.DeepCopy(),
-			Tolerations:              backupTemplate.Tolerations,
-			ToolImage:                backupTemplate.ToolImage,
-			ImagePullSecrets:         backupTemplate.ImagePullSecrets,
-			ServiceAccount:           backupTemplate.ServiceAccount,
-			CleanPolicy:              backupTemplate.CleanPolicy,
-			PriorityClassName:        backupTemplate.PriorityClassName,
-			CalcSizeLevel:            backupTemplate.CalcSizeLevel,
-			AdditionalVolumes:        backupTemplate.AdditionalVolumes,
-			AdditionalVolumeMounts:   backupTemplate.AdditionalVolumeMounts,
+			Mode:                                pingcapv1alpha1.BackupModeVolumeSnapshot,
+			Type:                                pingcapv1alpha1.BackupTypeFull,
+			FederalVolumeBackupPhase:            pingcapv1alpha1.FederalVolumeBackupExecute,
+			ResourceRequirements:                *backupTemplate.ResourceRequirements.DeepCopy(),
+			Env:                                 backupTemplate.Env,
+			BR:                                  backupTemplate.BR.ToBRMemberConfig(clusterMember.TCName, clusterMember.TCNamespace),
+			StorageProvider:                     *backupTemplate.StorageProvider.DeepCopy(),
+			Tolerations:                         backupTemplate.Tolerations,
+			ToolImage:                           backupTemplate.ToolImage,
+			ImagePullSecrets:                    backupTemplate.ImagePullSecrets,
+			ServiceAccount:                      backupTemplate.ServiceAccount,
+			CleanPolicy:                         backupTemplate.CleanPolicy,
+			PriorityClassName:                   backupTemplate.PriorityClassName,
+			CalcSizeLevel:                       backupTemplate.CalcSizeLevel,
+			AdditionalVolumes:                   backupTemplate.AdditionalVolumes,
+			AdditionalVolumeMounts:              backupTemplate.AdditionalVolumeMounts,
+			VolumeBackupInitJobMaxActiveSeconds: backupTemplate.VolumeBackupInitJobMaxActiveSeconds,
```

Comment:
You can set this value in the code block of `if initialize {}` below.

Reply:
done